README-VN.md

```
@@ -88,55 +88,62 @@ Astrocelerate có bản commit đầu tiên vào ngày 28 tháng 11 năm 2024. T
 
 
 ## Kho ảnh
-Các ảnh chụp màn hình sau đây ghi lại quá trình phát triển của Astrocelerate (Format ngày: ngày/tháng/năm).
+Các ảnh chụp màn hình sau đây ghi lại quá trình phát triển của Astrocelerate (Format ngày: YYYY-MM-DD).
 
-### 17/12/2025
+### 2026-04-12
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)